testsuite/forge-cli/src/main.rs

```diff
@@ -500,7 +500,7 @@ fn single_test_suite(
     let single_test_suite = match test_name {
         // Land-blocking tests to be run on every PR:
         "land_blocking" => land_blocking_test_suite(duration), // to remove land_blocking, superseeded by the below
-        "realistic_env_max_load" => realistic_env_max_load_test(duration, test_cmd, 7, 5),
+        "realistic_env_max_load" => pfn_const_tps(duration, true, true),
```

Comment:
You probably want to revert this change before landing?

Reply:
Done!

Reply:
Oops, now really reverted.